php/utils.php

```diff
@@ -100,8 +100,8 @@ function isPathAbsolute($path) {
  * @return [boolean] $is_email True if $email is in email address format
  */
 function isValidEmail($email) {
-  $is_email = filter_var($email, FILTER_VALIDATE_EMAIL);
-  return (boolean)$is_email;
+  $is_email = !is_bool(filter_var($email, FILTER_VALIDATE_EMAIL));
```

Comment:
I think FILTER_VALIDATE_EMAIL will return true/false only (and not null), so I don't think we need to wrap it with is_bool. http://php.net/manual/en/filter.filters.validate.php